STM documentations and thingy fix (#64)
AVSystem/developer-zone: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md

- The STM32L496G-DISCO/MONARCH board with a USB cable.
- Installed **STM32CubeIDE**.
- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
- A user with access to the Coiote IoT Device Management platform.

## Prepare binaries
### Use an already built binary

To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).

> In this case we are going to use the Anjay-freertos-client-STM32L496G-MONARCH.bin file which after downloaded must be added to STM32L496G-DISCO/MONARCH board folder.

Reviewer: ```suggestion
In this case we are going to use the Anjay-freertos-client-STM32L496G-MONARCH.bin file which after being downloaded must be added to STM32L496G-DISCO/MONARCH board folder.
```

> In this case we are going to use the Anjay-freertos-client-STM32L496G-MONARCH.bin file which after being downloaded must be added to STM32L496G-DISCO/MONARCH board folder.

Reviewer 2: and here

    3. In the **Device credentials** step:
         - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
             ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
         - In the **Security mode** section, select the **PSK (Pre-shared Key)** mode:
              - In the **Key identity** field, type `test_device`
              - In the **Key** field, type the shared secret used in the device-server authentication.
    4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
    5. In the **Connect your device** step, wait for the board to connect.
    6. Click **Go to device** to see your added device dashboard.

    !!! tip

>         LwM2M Server URI, endpoint name and other informations you can read from **configuration** tab.

Reviewer: ```suggestion
        LwM2M Server URI, endpoint name and other informations you can be found in the **configuration** tab.
```
